spec/features/daily_digest_spec.rb

```diff
@@ -1,12 +1,12 @@
 RSpec.describe "Daily digests", type: :request do
-  let(:list_one_topic_id) { "0eb5d0f0-d384-4f27-9da8-3f9e9b22a820" }
+  let(:list_one_document_collection_id) { "0eb5d0f0-d384-4f27-9da8-3f9e9b22a820" }
   let(:list_two_taxon_id) { "6416e4e0-c0c1-457a-8337-4bf8ed9d5f80" }
 
   let(:subscriber_list_one) do
     create_subscriber_list(
       title: "Subscriber list one",
       links: {
-        topics: { any: [list_one_topic_id] },
+        document_collections: { any: [list_one_document_collection_id] },
```

Comment:
Oh no, sorry Mat. You're right in that `document_collections` is a valid `links` key, but a subscriber list would only have this key if it also had a content id. It's not possible to have a link to a document collection without a content id. There are docs [here](https://github.com/alphagov/email-alert-api/blob/main/docs/matching-content-to-subscriber-lists.md#uuid-field) which I added last year - that I now realise are really not clear. So I will take an action to improve those! How about you pick `organisations` instead?

Reply:
Fixed. I've used organisations in most places. In a few o them I've used people or countries, as discussed.